kafka_utils/kafka_check/commands/offline.py

```diff
@@ -42,18 +43,21 @@ def run_command(self):
         )
 
         errcode = status_code.OK if not offline else status_code.CRITICAL
-        out = _prepare_output(offline, self.args.verbose)
+        out = _prepare_output(offline, self.args.verbose, self.args.head)
         return errcode, out
 
 
-def _prepare_output(partitions, verbose):
+def _prepare_output(partitions, verbose, head_limit):
```

Comment:
Would it make sense to have some kind of generic prepare_output function that can be re-used? Would that add too much complication?

Reply:
I was thinking about it multiple times before. Have a look at _prepare_output() - 80-90% of it is transformation from internal-representation to string. That transformation is very command-specific, so it won't go into shared place. Having generic function will make everything way harder to read, probably it will safe 10 lines by price of simplicity. Maybe with some tricks we can make it nicer. However, it's command line tools, I don't we should be perfect here.

Reply:
Sounds good.